### ISSUE TITLE
Typo in .github/actions/drainpipe/pantheon/review/action.yml

### ISSUE THREAD

**Issue by tess-ten7** [Lullabot/drainpipe#356]
During a PantheonReviewApps build, with no 'update' task in the project's Taskfile.yml, the following error occurs:

```
Failed to execute command ./vendor/bin/task -l | grep '* update: '): exit status 2
```

The error appears to be in .github/actions/drainpipe/pantheon/review/action.yml where there's an unnecessary close paraen:

```
elif drainpipe_exec "./vendor/bin/task -l | grep '* update: ')"; then
```

**Comment by quicksketch** [comment]
I ran into this same issue using the `update` Pantheon action, see #601. The PR I filed at https://github.com/Lullabot/drainpipe/pull/602 should fix this issue and #601.

**Comment by quicksketch** [comment]
Fixed in https://github.com/Lullabot/drainpipe/pull/602. Thank you @tess-ten7!
